examples/visualization/plot_publication_figure.py

```diff
@@ -69,6 +69,7 @@
 # 255, 255)`` encodes a white pixel, so we'll detect any pixels that differ
 # from that:
 
+screenshot[(screenshot == 0)] = 255
 nonwhite_pix = (screenshot != 255).any(-1)
```

Comment:
or maybe better, don't do the above and instead do
```suggestion
nonwhite_pix = (screenshot[..., :3] != 255).any(-1)
```

Reply:
If you go with this approach, you should also change the explanatory text above this code block

Reply:
i.e., explain why using `:3` (to look only at RGB and ignore alpha channel)